hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/webapp/AbstractRESTRequestInterceptor.java

```diff
@@ -86,4 +92,32 @@ public RESTRequestInterceptor getNextInterceptor() {
     return this.nextInterceptor;
   }
 
+  private void setupUser(final String userName) {
+    try {
+      /**
```

Comment:
This looks like the method javadoc, let's make it so.

Reply:
Thanks for your help reviewing the code, I will fix it.